platform/core/Makefile

```diff
@@ -31,8 +31,11 @@ swift_flatbuffer_models = $(patsubst $(flatbuffer_models_dir)/%, $(swift_flatbuf
 ifeq ($(OS),Windows_NT)
 	flatc_download_url = $(flatbuffers_releases)/Windows.flatc.binary.zip
 	flatc := $(flatc_dir)/flatc.exe
+	# Need on Windows, to allow interpreting escaping characters
+	echo := echo -e
 else
 	flatc := $(flatc_dir)/flatc
+	echo := echo
```

Comment:
You probably don't need this, right? $(echo) will not evaluate to itself?
If it is necessary, something less confusing like `osPrint` or `thisEcho` or `os_echo` might be an option.